lms/static/images/icon-cert-white.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12.85 9.22">
+    <defs>
+        <style>.cls-1{fill:#fff;}</style>
```

Comment:
@AlasdairSwan agreed, I'm trying to do that so I can use this SVG instead of the green PNG on the left, too. How would you suggest we color SVGs used as background images? I've been reading [this](http://codepen.io/noahblon/post/coloring-svgs-in-css-background-images) and was thinking about using a CSS filter. It would be great to use a CSS mask, but they don't seem well supported.